docs/src/Provider.doc.js

```diff
@@ -191,10 +191,11 @@ function OnNavigation() {
 `}
       />
       <MainSection.Card
+        title="Examples above from top to bottom: (1) Callout, (2) Upsell, (3) ActivationCard"
```

Comment:
Titles are great, thank you! Side note: I'm gonna that with Annie about this styling though, I'm thinking for large examples the title needs to stay above

Reply:
agreed